Source/KSCrash-Tests/KSCrashDoctor_Tests.m

```diff
@@ -0,0 +1,24 @@
+#import <XCTest/XCTest.h>
+
+#import "KSCrashDoctor.h"
+
+@interface KSCrashDoctor_Tests : XCTestCase @end
+
+
+@implementation KSCrashDoctor_Tests
+
+- (NSDictionary<NSString *, id> *)_crashReportAsJSON:(NSString *)filename
+{
+    NSURL *url = [[NSBundle bundleForClass:self.class] URLForResource:filename withExtension:@"json"];
```

Comment:
Bundle resources in SPM tests behave a little differently. Please see this code. 
https://github.com/kstenerud/KSCrash/blob/cc4eff4dbbcf987e196edbb8ae7441b2ead91512/Tests/KSCrashRecordingTests/KSCrashReportFixer_Tests.m#L12-L16
Perhaps we should move this code into the `KSCrashTestTools` module, as it now needs to be shared.